src/app/wallets/index.ts

```diff
@@ -21,3 +21,10 @@ export const getWallet = async (walletId: WalletId) => {
   const wallets = WalletsRepository()
   return wallets.findById(walletId)
 }
+
+export const getWalletByPublicId = async (
+  walletPublicId: WalletPublicId,
+): Promise<Wallet | ApplicationError> => {
+  const wallets = WalletsRepository()
+  return wallets.findByPublicId(walletPublicId)
+}
```

Comment:
Why this wrapper around `findByPublicId`?

Reply:
Servers should only access app layer (not service layer) also, as mentioned in main comment, was not refactored in main use case because will be updated with the new authorization schema